benchmark/kvbench/runtime/etcd_rt.py

```
@@ -123,15 +151,14 @@ def barrier(self, ranks: Optional[List[int]] = None, timeout_sec=600):
             while not self.client.replace(
                 key, int_to_bytes(len(ranks)), int_to_bytes(0)
             ):
+                current_val = self._get_int_val(key)
                 if timeout_sec and time.time() - start_time > timeout_sec:
+                    missing_ranks = (
+                        [r for r in ranks[current_val:]] if current_val else ranks[1:]
```

Ruff (0.15.15)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)